src/GraphQL/GraphQLExtensions.cs

```diff
@@ -491,5 +491,63 @@ public static IValue AstFromValue(this object value, ISchema schema, IGraphType
 
             throw new ExecutionError($"Cannot convert value to AST: {serialized}");
         }
+
+
+        public static bool References(this ISchema schema, string typeName)
```

Comment:
I'm not sure about the name. This method works not only for scalars, but for all types in general. Theoretically, it is possible to make such a schema in which there will be a "independent" type.

Reply:
Renamed to `Supports` due to spec: 
> A schema is defined in terms of the types and directives it supports as well as the root operation types for each kind of operation